src/client/api.ts

```diff
@@ -3,8 +3,7 @@
 
 'use strict';
 
-import { Disposable, Event, Uri } from 'vscode';
-import { NotebookCell, NotebookCellRunState } from '../../types/vscode-proposed';
```

Comment:
Since our extension can use proposed in stable & insiders, we can just import types directly from VS Code.